### PR TITLE
Add StateChart, Play, PlayAction models

### PR DESCRIPTION
**Pull request by rib3**
Add StateChart, Play, PlayAction models

* [TOURNEY-1685](https://sportngin.atlassian.net/browse/TOURNEY-1685)

### CODE REVIEW

**Comment by sportngin-code-climate** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/52403ace89af7e683c02b17d/compare/state-chart">analyzed this pull request</a>.

File: src/sports/playAction.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+   */
+
+  var PlayAction = SportsModel.extend({
+
+    fetch: function(options, callback) {
+      return Super.fetch.call(this, this.urlById(), options, callback)
+    },
+
+    save: function(options, callback) {
+      var url = PlayAction.urlRoot() + (this.id ? '/' + this.id : '')
```
Comment (jessehouchins) [inline src/sports/playAction.js]: can this use urlById as well?

File: src/sports/stateChart.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+   *
+   * @param {Object} attr
+   * @param {Object} options
+   * @api public
+   */
+
+  var StateChart = SportsModel.extend({
+
+    fetch: function(options, callback) {
+      var url = StateChart.urlRoot() + '/' + this.id
```
Comment (jessehouchins) [inline src/sports/stateChart.js]: urlById ?

**Comment by jessehouchins** [comment]
Just the two cases where it looks like you could use `urlByUd()` but nothing else. CR :+1:

**Comment by rib3** [comment]
Good call, I added that helper late in the process and must have forgotten to update those two

**Pull request merged**